common/v2/features/SendAssets/components/fields/GasPriceSlider.tsx

```diff
@@ -59,7 +70,13 @@ export default class SimpleGas extends Component<Props> {
               <div className="GasPriceSlider-slider">
                 <SliderWithTooltip
                   {...field}
-                  onChange={e => form.setFieldValue('gasPriceSlider', e)}
+                  onChange={e => {
+                    this.setState({ gasPrice: e.toString() });
+                  }}
```

Comment:
Since this component is in a form I believe we can make a controlled component and remove the inner state. It can receive the value from the parent, and update the parents state with the event handler.

Reply:
I'm not sure I understand this. We want the inner state to reduce render calls in the form, as everytime we update the form it re-renders?